config/config.yaml

```diff
@@ -1,20 +1,21 @@
 project_code: aws-workflows-nextflow-infra
-profile: {{ var.profile | default() }}
-region: {{ var.region | default("us-east-1") }}
+profile: {{var.profile | default()}}
```

Comment:
any reason for all these formatting changes?

Reply:
My auto-formatter got me, reverted in the latest commit.